### ISSUE TITLE
Training From Extracted Feature Vector

### ISSUE THREAD

**Issue by seewalker** [BVLC/caffe#690]
I have a feature vector produced by running `extract_features` on a pretrained model and my data. I want to train a neural network using that feature vector as the input blob. What layer type would that data be considered? Or, if I'm on the wrong train, how can I train a network with the result of `extract_features` as my starting point?

I've tried making the first layer of my network a DATA layer like so:

```
layers {
    name: "data"
    type: DATA 
    top: "data"
    top: "label"
    data_param {
        source: "/home/a/learning/train_feature_vector"
        batch_size: 50
        mirror: false
    }
}
```

and having the next layer consider this layer, "data", its `bottom`. I've also tried making this layer have a  `image_data_param` instead of a `data_param`, but with no success. Is there something deeply wrong with my approach, or am I missing some detail?

In case it matters, the layer being extracted is an inner product layer (with in-place relu and dropout layers) and the first layer after this "data" layer is also such an inner product layer.
